src/runtime/components/forms/Slider.vue

```diff
@@ -0,0 +1,244 @@
+﻿<template>
+  <div :class="ui.wrapper">
+    <input
+      :id="name"
+      ref="input"
+      :name="name"
+      :value="modelValue"
+      :disabled="disabled"
+      type="range"
+      :class="inputClass"
+      v-bind="$attrs"
+      @input="onInput"
+      @focus="$emit('focus', $event)"
+      @blur="$emit('blur', $event)"
+    >
+    <slot />
+  </div>
+</template>
+
+<script lang="ts">
+import { ref, computed, onMounted, defineComponent } from 'vue'
+import type { PropType } from 'vue'
+import { defu } from 'defu'
+import { classNames } from '../../utils'
+import { useAppConfig } from '#imports'
+// TODO: Remove
+// @ts-expect-error
+import appConfig from '#build/app.config'
+
+export default defineComponent({
+  props: {
+    modelValue: {
+      type: [String, Number],
+      default: ''
```

Comment:
I think modelValue should be a Number with a default value of 0, right?

Reply:
Yep have fixed